sofl/dialogs/install_dialog.py

```
@@ -430,189 +418,196 @@ def on_install_clicked(self, button):
         """Generates a unique game ID for online-fix games
-        
+
         Returns:
             str: Generated game ID
         """
         source_id = "online-fix"
         numbers = [0]
         prefix = "online-fix_"
-        
+
         for game_id in shared.store.source_games.get(source_id, set()):
```

Pylint (3.3.7)

[error] Module 'sofl.shared' has no 'store' member

(E1101)